### ISSUE TITLE
Error 'apksigner' returned with error #1

### ISSUE THREAD

**Issue by YurumiBits** [godotengine/godot#64992]
### Godot version

3.5.stable.official [991bb6ac7]

### System information

Windows10, GLES3, AMD RYZEN 3 3700x, AMD R9 380

### Issue description

Hi, i'm trying to export my project to android platform. I do have seen multiple tutorials and read multiple documentations files about setting it up. Below my setup:

Godot:
![immagine](https://user-images.githubusercontent.com/20691297/187081794-71a7cc9f-1014-4db1-a78c-bf238d18d759.png)
![immagine](https://user-images.githubusercontent.com/20691297/187081895-0d4c832d-cbde-434b-9cd2-e615365e085c.png)

JDK:
![immagine](https://user-images.githubusercontent.com/20691297/187081846-46ca16b1-c368-4887-a7c7-c1f661222d8c.png)

Android SDK:
![immagine](https://user-images.githubusercontent.com/20691297/187081935-ae828962-7878-4633-a961-7c1533bff8d5.png)
![immagine](https://user-images.githubusercontent.com/20691297/187081949-bd4ee67e-f842-4900-994e-446da70638fb.png)

That's the output when i try to export the project:
![immagine](https://user-images.githubusercontent.com/20691297/187081989-70a1f4e5-adf4-4fb5-940b-09a9e3166340.png)
![immagine](https://user-images.githubusercontent.com/20691297/187082026-afe301d1-3562-4ca4-a884-b00a60c87b90.png)

### Steps to reproduce

Create a basic project and try to export is for android platform

**Comment by Calinou** [comment]
Are you launching Godot through Steam? See https://github.com/godotengine/godot/issues/62768.

**Comment by YurumiBits** [comment]
> Are you launching Godot through Steam? See #62768.

Nope, no steam

**Comment by YurumiBits** [comment]
Maybe I know what's wrong guys!
First of all i go to the android's build-tools folder
`Your_Android_SDK_FOLDER\build-tools\30.0.3\`
then i found the 'apksigner.bat' file, opened it and started to change, one at the time the exit codes, until I found what's the error that i get, see pic below
 
![immagine](https://user-images.githubusercontent.com/20691297/187095488-05d64645-4f38-40ae-a02d-b556b4688a8d.png)

Long story short:
`ERROR: JAVA_HOME is set to an invalid directory: %JAVA_HOME% 
Please set the JAVA_HOME variable in your environment to match the location of your Java installation.`

I try to solve this problem then update you. Sorry for my bad english guys

**Comment by YurumiBits** [comment]
> Maybe I know what's wrong guys! First of all i go to the android's build-tools folder `Your_Android_SDK_FOLDER\build-tools\30.0.3\` then i found the 'apksigner.bat' file, opened it and started to change, one at the time the exit codes, until I found what's the error that i get, see pic below
> 
> ![immagine](https://user-images.githubusercontent.com/20691297/187095488-05d64645-4f38-40ae-a02d-b556b4688a8d.png)
> 
> Long story short: `ERROR: JAVA_HOME is set to an invalid directory: %JAVA_HOME% Please set the JAVA_HOME variable in your environment to match the location of your Java installation.`
> 
> I try to solve this problem then update you. Sorry for my bad english guys

Yeah guys, it was a bad case of JAVA_HOME path gone wrong. Close this issue and sorry for bothering you. Have a nice day! YUPPIE!!!

**Comment by Clamydeef** [comment]
So what was wrong? I have the same problem and I can't solve it

**Comment by Clamydeef** [comment]
![image](https://user-images.githubusercontent.com/94852696/219963856-e742444d-61df-4da1-8210-ace0c8d84d48.png)
I followed the same path and found that the error occurs at this location. It then goes to : note and throws an error from "shift / 1" where "set params". Java experts, tell me, what is causing the error?
